pkg/plugins/qingcloud/provider.go

```diff
@@ -22,6 +26,7 @@ type Provider struct {
 }
 
 func (p *Provider) ParseClusterConf(versionId, conf string) (*models.ClusterWrapper, error) {
+	cluster := app.Cluster{}
```

Comment:
Can app.Cluster be changed to other name, like clusterTmpl or clusterJsonTmpl. 
Cluster is easy to be misunderstand.

Reply:
done